.run/BC7 Debug tests.run.xml

```diff
@@ -1,5 +1,5 @@
 <component name="ProjectRunConfigurationManager">
-  <configuration default="false" name="Debug tests" type="GradleRunConfiguration" factoryName="Gradle">
+  <configuration default="false" name="BC7 Debug tests" type="GradleRunConfiguration" factoryName="Gradle">
```

Comment:
Renamed and added run configurations for Android Studio to be able to run tests for a specific flavor more easily.